### PR TITLE
follow user 

### PR DESCRIPTION
**Pull request by dangthihoa**
Related Ticket: https://edu-redmine.sun-asterisk.vn/issues/28130
What: User can follow/unfollow other user

Screenshot from 2020-08-26 10-54-53

### CODE REVIEW

**Review by HieuNT-Sun-Asterisk — APPROVED** [review]

**Review by Atanna128 — APPROVED** [review]

**Comment by dangthihoa** [comment]
ready

**Comment by dangthihoa** [comment]
ready

**Comment by dangthihoa** [comment]
ready

File: travel/views.py [hunk 1]
```diff
@@ -32,17 +32,39 @@ def front_page(request):
     # Render the HTML template index.html with the data in the context variable
     return render(request, 'front_page.html', context=context)
 
+@login_required
+def follow(request,pk):
+    tuser = get_object_or_404(User, pk=pk)
+    temp = 0
```
Comment (thaodtd-0665) [inline travel/views.py]: đặt lại tên biến cho rõ ràng nha em, vd: is_follower, ...

Comment (dangthihoa) [reply travel/views.py]: vang a

**Comment by dangthihoa** [comment]
ready

**Review by thaodtd-0665 — APPROVED** [review]

**Pull request merged**